### ISSUE TITLE
Panic on score submit attempt in v1.3.0

### ISSUE THREAD

**Issue by wermipls** [GrooveStats/gslauncher#11]
Tested on amd64 build of the launcher. Doesn't happen on 1.2.0 or 1.2.1.

Windows 10 x64, Outfox Alpha date-20211001, Simply Love 5.1.0 (with [some modifications](https://github.com/wermipls/Simply-Love-SM5), but I don't think it matters).

Log excerpt:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x50 pc=0x781258]

goroutine 24 [running]:
github.com/GrooveStats/gslauncher/internal/groovestats.(*Client).scoreSubmit(0xc0002ecc60, 0xc0001a2030, 0xc0004a7770, 0xc0004a7780, 0xc0004a7790)
        /data/internal/groovestats/client.go:268 +0x5b8
github.com/GrooveStats/gslauncher/internal/groovestats.(*Client).ScoreSubmit(0xc0002ecc60, 0xc0001a2030, 0xc0004a7830, 0xc0004a7840, 0xc0004a7850)
        /data/internal/groovestats/client.go:211 +0x45
github.com/GrooveStats/gslauncher/internal/session.(*Session).processRequest(0xc0002ecd80, 0xaec180, 0xc0001a2030)
        /data/internal/session/session.go:178 +0x26b
github.com/GrooveStats/gslauncher/internal/session.(*Session).startIpc.func1(0xc000052600)
        /data/internal/session/session.go:90 +0x8e
created by github.com/GrooveStats/gslauncher/internal/session.(*Session).startIpc
        /data/internal/session/session.go:97 +0x250
```

Another example:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x50 pc=0x781258]

goroutine 23 [running]:
github.com/GrooveStats/gslauncher/internal/groovestats.(*Client).scoreSubmit(0xc0005686f0, 0xc000318000, 0x5900000059000000, 0x5900000059000000, 0x5900000059000000)
        /data/internal/groovestats/client.go:268 +0x5b8
github.com/GrooveStats/gslauncher/internal/groovestats.(*Client).ScoreSubmit(0xc0005686f0, 0xc000318000, 0x5900000059000000, 0x5900000059000000, 0x5900000059000000)
        /data/internal/groovestats/client.go:211 +0x45
github.com/GrooveStats/gslauncher/internal/session.(*Session).processRequest(0xc000568840, 0xaec180, 0xc000318000)
        /data/internal/session/session.go:178 +0x26b
github.com/GrooveStats/gslauncher/internal/session.(*Session).startIpc.func1(0xc0003944e0)
        /data/internal/session/session.go:90 +0x8e
created by github.com/GrooveStats/gslauncher/internal/session.(*Session).startIpc
        /data/internal/session/session.go:97 +0x250
```

**Comment by wermipls** [comment]
And also. I'm playing on P2. Which does seem to be relevant in this case, considering the P2 code tries to reference P1 data LOL

https://github.com/GrooveStats/gslauncher/blob/66376d2f6ce240757fc98dee96a03a3d8b737185/internal/groovestats/client.go#L264-L269

**Comment by natano** [comment]
Oh god, I'm an idiot :facepalm: I'll release a hotfix in a bit.

**Comment by natano** [comment]
This is fixed in [v1.3.1](https://github.com/GrooveStats/gslauncher/releases/tag/v1.3.1).
